modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/transaction/TransactionalSourceTestCase.java

```diff
@@ -122,31 +147,65 @@ private void validate(CheckedSupplier<Boolean> validation) {
     new PollingProber(10000, 100).check(new JUnitLambdaProbe(validation));
   }
 
-  private void validateCommittedTransaction(TestTransactionalConnection connection) {
-    assertThat(connection.isTransactionBegun(), is(true));
-    assertThat(connection.isTransactionCommited(), is(true));
-    assertThat(connection.isTransactionRolledback(), is(false));
-  }
-
-  private void validateRolledBackedTransaction(TestTransactionalConnection connection) {
-    assertThat(connection.isTransactionBegun(), is(true));
-    assertThat(connection.isTransactionCommited(), is(false));
-    assertThat(connection.isTransactionRolledback(), is(true));
-  }
-
-  private void validateNonTxConnection(TestTransactionalConnection connection) {
-    assertThat(connection.isTransactionBegun(), is(false));
-    assertThat(connection.isTransactionCommited(), is(false));
-    assertThat(connection.isTransactionRolledback(), is(false));
+  private void validateCommittedTransaction(Object connection) {
```

Comment:
It is fine to parameterize this test, but if on each test there is a need to ask if we are in one case or the other, I think we are not gaining much from the parametetization